protocol/message_validator.go

```diff
@@ -2,17 +2,52 @@ package protocol
 
 import (
 	"errors"
+	"math/big"
 	"strconv"
 	"strings"
 
 	"github.com/status-im/status-go/protocol/protobuf"
+	"github.com/status-im/status-go/protocol/v1"
```

Comment:
How do I import `MembershipUpdateMessage` otherwise? I tried removing it, but without it wont' compile, is there another way? (in `messenger.go` we import I  believe)

Reply:
Ah sorry. It is needed. I though I deleted this comment...